### ISSUE TITLE
No video in Safari on either Mac OS or iOS

### ISSUE THREAD

**Issue by k-ahn** [aws/amazon-chime-sdk-js#812]
iOS Version: 13.6
Mac OS Version: 10.13.6
SDK Version: "amazon-chime-sdk-js": "^1.20.2",

**NB:** I've edited this issue to remove an incorrect hypothesis that the issue was related to deviceId, then added the answer.

**ISSUE**
I've built a React JS app using the SDK and video does not work in Safari. The LED flashes on and then disappears. The summary is:
- Video works on Chrome on Mac OS or Windows 
- Video does not work in Safari on either Mac OS or iOS
- Video in the browser demo does work in Safari on Mac OS (i couldn't test it on iOS)

**UPDATE:** 
I found the issue:
- The demo app starts audio input and video at the same time. This works.
- My app starts audio input, audio output and video at the same time. This does not work
- After logical deduction i found running the following in the same try block was causing the issue:
```
await session.audioVideo.chooseVideoInputDevice(videoDeviceId)
await session.audioVideo.chooseAudioOutputDevice(audioOutputDeviceId)
```

So in summary:
- WORKS: Starting video and audio input at the same time
- WORKS: Starting audio input and audio output at the same time
- DOES NOT WORK: Starting video and audio output at the same time 

The fix was therefore to start video first, then audio input and output.  Hope this helps someone.
